cypress/fixtures/tpl/docs.index.html

```diff
@@ -109,4 +110,4 @@
   <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
can you add Line here

Reply:
I don't know how to change it... 
I've tried several times and it still looks like this.

Reply:
can you create a new line manually and turn off the auto format on save in your editor.

Reply:
Is it like this now?

Reply:
I think it was the original that didn't have a line feed, that was wrong...
![image](https://user-images.githubusercontent.com/33931153/93661889-14d49e80-fa8e-11ea-8219-cdd4d3212256.png)

Reply:
Interesting. let me have a try.

Reply:
I realize this is fine that it is the origin file lost the new line.